modular_skyrat/modules/ashwalkers/code/buildings/railroad.dm

```diff
@@ -70,6 +70,7 @@
 /obj/vehicle/ridden/rail_cart/examine(mob/user)
 	. = ..()
 	. += span_notice("<br><b>Alt-Click</b> to attach a rail cart to this cart.")
+	. += span_notice("<br>Filling it with <b>10 sand</b> will allow it to be used as a planter!")
```

Comment:
The `. += something` already makes it skip a line between each element of that list. I totally agree with separating the notices from the rest though, yeah.
```suggestion
	. += ""
	. += span_notice("<b>Alt-Click</b> to attach a rail cart to this cart.")
	. += span_notice("Filling it with <b>10 sand</b> will allow it to be used as a planter!")
```

Reply:
I would rather not do this suggestion. The br does what you are suggesting.